### DISCUSSION TITLE
problem during  to registration

### DISCUSSION THREAD

**Discussion by sohaib929** [dashingsoft/pyarmor#1591] [Q&A]
I purchased a group license, and unfortunately every time I try to complete the registration process, an error occurs. I sent an email to the entity, and unfortunately I was informed that I should present the problem here.
Please help in solving this problem
![image](https://github.com/dashingsoft/pyarmor/assets/57887405/a535d11c-ce0a-47ed-b532-567a553b93d4)

**Comment by jondy** [comment]
Please first read this guide
https://pyarmor.readthedocs.io/en/latest/how-to/register.html#using-group-license

**Reply by sohaib929** [reply]
Thanks for your reply
I read all the instructions on the site and followed them as written, and the problem still exists

**Comment by jondy** [comment]
Could you report an issue according the issue template?
https://github.com/dashingsoft/pyarmor/issues/
A good report should have

A clear title
Reproduced steps
Actual results
Expected results
